staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -6746,3 +6747,100 @@ func TestUnknownField(t *testing.T) {
 		}
 	}
 }
+
+func TestStrategicMergePatchUnstructured(t *testing.T) {
```

Comment:
When writing this test, I didn't notice we already had a unit test at `staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go`. Anyway, it doesn't hurt to have extra tests at different layers.